examples/rust/request_response/README.md

```diff
@@ -0,0 +1 @@
+# Request-Response
```

Comment:
Is it intentional that the readme contains only one line?

Reply:
Yes since the example is not yet finished. As soon as the example demonstrates the full request-response functionality the readme will be filled out.